problems/1.types+operators.js

```diff
@@ -2,15 +2,19 @@
 * @param {any} val - The input value
 * @returns {boolean} - Is val a number
 */
-function isNumber() {
-
+function isNumber (val = 15) {
+    if(val => 1)
+      return "val is a number";
```

Comment:
This should return a boolean. YOu're returning a number. Think about using typeof to help you determine, to see if its' greater than 1. Numbers less than 1 are still numbers. REmember to have curly brackets with your if statements.